uint8 indiID
---
uint8 res
float32 indiData